lib/mix/lib/mix/tasks/help.ex

```diff
@@ -56,8 +59,9 @@ defmodule Mix.Tasks.Help do
     tasks = Enum.map(load_tasks(), &Mix.Task.task_name/1)
 
     aliases =
-      Mix.Project.config()[:aliases]
-      |> Enum.map(fn {k, _} -> Atom.to_string(k) end)
+      Enum.map(Mix.Project.config()[:aliases], fn {alias_name, _} ->
+        Atom.to_string(alias_name)
+      end)
```

Comment:
Why not use `aliases = load_aliases()` here too?

Reply:
I mean that and then `Enum.sort(Map.to_list(aliases) ++ tasks)`.

Reply:
`Map.to_list(aliases)` won't work because aliases is `[{alias1, taks1}, {alias2, task2}, ...]`.
If we want to use `load_aliases()`, we might need to do

```elixir
    aliases = Enum.map(load_aliases(), fn {alias_name, _} -> alias_name end)

    for info <- Enum.sort(aliases ++ tasks) do
      Mix.shell().info(info)
    end
```

At that point, we might as well use `Mix.Project.config()[:aliases]` directly, so we don't have to iterate twice.

Reply:
Oh I missed that it was just the alias name, sorry!